### DISCUSSION TITLE
Documentation PR's?

### DISCUSSION THREAD

**Discussion by Fxplorer** [fsprojects/Avalonia.FuncUI#402] [General]
Am I totally missing the boat here? I see the docs are on gitbook, is there a repo to the source? I did just find the wiki actually, but that looks to have not been updated in a while. Is it not of concern anymore?
 
I am trying to wrap my head around Avalonia and FuncUI and app development in general, and while I have wanted to make apps in F# (well, any platform really) for almost 14 years now, still have not made one, but I keep trying.

**Comment by Numpsy** [comment]
According to https://github.com/fsprojects/Avalonia.FuncUI/issues/260 you need permissions to update the docs, but I haven't tried making any doc changes myself in that system.
(Also refs #398 as there might be a doc update needed there)

**Reply by JaggerJo** [reply]
@Fxplorer It's now possible to contribute to the docs 🎉

Changes are automatically synced..

https://github.com/fsprojects/Avalonia.FuncUI/tree/master/docs

**Reply by Fxplorer** [reply]
I won't guarantee anything, but I am trying to grasp all of this and I am finding that there many controls missing from the current docs. Sometimes coming in from the newbie angle gives rise to questions that might be helpful in the docs :)

Thank you, hopefully I can contribute!
